wg-releases/README.md

```diff
@@ -66,6 +66,12 @@ This is done primarily to ensure that there are no open avenues of compromise fo
 
 Meeting notes may be viewed in [meeting-notes](meeting-notes).
 
-## Feature Backport Requests for Major Versions in Beta
-
-With our 12-week major release cadence, we are _not_ considering feature backport requests after the Releases WG meeting of Week 3 of each beta cycle. Backport requests after Week 3 will only be considered if there is a very good reason. For backport requests please submit the link to the PR and reason for backport to the Releases WG agenda for consideration.
+## Feature Backport Requests
+Backporting non-breaking features to supported versions requires the approval of the Releases Working Group.  Any feature PR that is targeting branch(es) other than master should be labeled with a `pending-vote 🗳` label.  The approval process will work as follows:
+1. By requesting backport of a feature, the submitter of the PR is assuming responsibility for this feature on the requested branches.  If it causes breakage, you will be responsible for fixing the breakage. 
```

Comment:
Jumps from passive to active voice with "the submitter" to "you". Also, since we can't enforce that a volunteer fixes anything, might be better to explain in terms of things the WG can enforce:

> If a backport PR causes any breakage, it may be removed if the PR submitter does not fix the breakage.